mapadroid/ocr/screenPath.py

```diff
@@ -129,7 +129,10 @@ def __evaluate_topmost_app(self, topmost_app: str) -> Tuple[ScreenType, dict, in
         returntype: ScreenType = ScreenType.UNDEFINED
         global_dict: dict = {}
         diff = 1
-        if "AccountPickerActivity" in topmost_app or 'SignInActivity' in topmost_app:
+        if "KO: Could not read topmost app. Is screen off?" in topmost_app:
+            logger.error("RGC could not read topmost app. Is screen off?")
+            return ScreenType.ERROR, global_dict, diff
```

Comment:
We could probably just check for startsWith "KO:" since any "KO" really is an issue. Then again, that should probably be located in the Communicator itself :thinking: